src/Sarif.FunctionalTests/UpdateBaselines.ps1

```diff
@@ -42,6 +42,13 @@ function Build-Baselines($toolName)
       $sourceExtension = "fpr"
     }
 
+    if ($ToolName -eq "FxCopProject")
+    {
+      $sourceExtension = "fxcop"
+      $toolName = "FxCop"
```

Comment:
>FxCop [](start = 19, length = 5)

Rather than special-casing `FxCopyProject` as you do here and on line 73, would you consider placing the `FxCopProject` files in a separate same-named subdirectory of `ConverterTestData`?